src/pipecat/processors/aggregators/llm_response.py

```diff
@@ -351,24 +355,18 @@ async def push_aggregation(self):
                 # No interruption config - normal behavior (always push aggregation)
                 await self._process_aggregation()
 
-    def _should_interrupt_based_on_strategies(self) -> bool:
+    async def _should_interrupt_based_on_strategies(self) -> bool:
         """Check if interruption should occur based on configured strategies."""
-        if not self.interruption_strategies:
-            return False
 
-        # Check strategies one by one until first match
         for strategy in self.interruption_strategies:
-            if isinstance(strategy, MinWordsInterruptionStrategy):
-                if self._should_interrupt_min_words(strategy):
-                    return True
+            await strategy.append_text(self._aggregation)
+            result = await strategy.should_interrupt()
+            await strategy.reset()
+            if result:
+                return True
```

Comment:
In the changelog you mentioned:
- "If multiple strategies are provided, all of them must evaluate to true for the interruption to occur."

If I'm not mistaken, in this case, we're interrupting if any of them are true , right ? I don’t think that’s wrong. But if we choose to keep it like this maybe just fix the changelog.